docs/userguide/services.md

```diff
@@ -118,16 +118,20 @@ ICU provides the following [classes](datetime/index.md) to support calendars and
 time zones:
 
 *   [Calendar](datetime/calendar/index.md#calendar)
```

Comment:
Since this is link with an anchor, I think we'll need to remove the `.md` part, otherwise it'll be broken.
Same with the ones below...
(Idea: You could probably search and replace "`.md#`" with empty string.)

Reply:
Done. I've checked all "`.md#`" references.

Reply:
Thanks!